encoders/nlp/UniversalSentenceEncoder/tests/test_universalsentenceencoder.py

```diff
@@ -75,3 +77,23 @@ def test_save_and_load_config(mocker):
     encoder_loaded = BaseExecutor.load_config(encoder.config_abspath)
     assert encoder_loaded.model_url == encoder.model_url
     rm_files([encoder.config_abspath])
+
+
+@mock.patch('tensorflow_hub.KerasLayer', return_value=MockModule())
+def test_get_universal_sentence_encoder(mocker):
+    MODEL_ENCODER = 'https://tfhub.dev/google/universal-sentence-encoder/4'
+    metas = get_metas()
+    encoder = UniversalSentenceEncoder(metas=metas)
+    assert encoder.model_url == MODEL_ENCODER
+
+
+@mock.patch('tensorflow_hub.KerasLayer', return_value=MockModule())
+def test_get_universal_sentence_encoder_cmlm(mocker):
```

Comment:
Please add some encoding here asserting some property of the resulting embedding to make sure it is tested with preprocessors

Reply:
I see another problem...The workflow cmlm implies to select a result dictionary 'default'
 `english_embeds = encoder(preprocessor(english_sentences))["default"]`
but for the old code it is used 
`model.load()`
As far I know, It is necessary to split in different two use cases and it can not be reused how I pretend in the current PR. For me, it seems easier to create two different private functions for each case.

Reply:
About the test... I didn t get it.. do you want to add tests for the validation of ouput preprocessor, is it correct  ?

Reply:
I want to add test that the encoding also works when there is preprocessing.

Reply:
ok and  e2e test to verify that it is all working well. not problem.